### PR TITLE
added component tag name convention [line 225]

### PR DESCRIPTION
**Pull request by sourabhmandal**
I faced an Issue while creating a component tags in my app and later realised that i have to capitalise first letter of the word to make it work.

### CODE REVIEW

**Comment by sourabhmandal** [comment]
> Hi, @sourabhmandal. Thanks for making your 1st contribution to the Ember Guides! I'm sorry to hear that you ran into trouble with figuring out the component naming conventions in Ember.
> 
> I read the section [Create a UI Component](https://guides.emberjs.com/release/getting-started/quick-start/#toc_create-a-ui-component) and its neighboring sections to understand the context in which the `<PeopleList>` component was introduced.
> 
> I noticed that, further down from where you wanted to introduce a paragraph, the code example does show how to invoke the `<PeopleList>` component with title case.
> 
> > quick_start
> 
> In addition, since the paragraph that you suggested introduces a component called `<Navbar>`, which isn't used in other sections in the page, I'm hesitant to allow making this change.
> 
> ## Suggested solution
> I do think the Quick Start page, which may be one of the first few pages that a new Ember developer may visit, can better show how to call a component.
> 
> Can I suggest the following actions:
> 
> * Revert the change that you made
> * In `guides/release/quick-start.md`, replace "`PeopleList` component" with "`<PeopleList>` component" to prime the reader. The pattern of using angle brackets to denote a component is used in other places in the Ember Guides, so I thought this would make a good solution.
>   Note, there are 5 places to replace. Here is an example:
>   ```gfm
>   
>   Let's create a `PeopleList` component that we can use in multiple places to show a list of people.
>   
>   
>   Let's create a `<PeopleList>` component that we can use in multiple places to show a list of people.
>   ```
> * In `guides/release/quick-start.md`, update the code example to show calling `<PeopleList>` with multiple lines:
>   ```
>   {{!-- Currently --}}
>   <PeopleList @title="List of Scientists" @people={{@model}} />
>   
>   {{!-- Suggested change --}}
>   <PeopleList
>     @title="List of Scientists"
>     @people={{@model}}
>   />
>   ```
> * Backport the changes above to `guides/v3.15.0/quick-start.md` to `guides/v3.19.0/quick-start.md` as well.
> 
> I look forward to hear what you think of these suggestions. Feel free to ask for clarifications.
> 
> Thanks again for providing feedback of your learning experience!

**Comment by sourabhmandal** [comment]
Hi @ijlee2  I agree with your suggested changes and have made changes accordingly.

Additionally, I have added following lines to make it clear that ember uses only title cased components to save the hassle of figuring it out themselves, as the docs doesnt specify to use camel case only nor it shows any error on running. It simply doesnt work.

```
<!-- currently -->
Note that we've changed the title from a hard-coded string ("List of Scientists") to `{{@title}}`. 
The `@` indicates that `@title` is an argument that will be passed into the component, 
which makes it easier to reuse the same component in other parts of the app we are building.

We've also renamed scientist to the more-generic person, decreasing the coupling of our 
component to where it's used.

<!-- Changed -->
Note that we've changed the title from a hard-coded string ("List of Scientists") to `{{@title}}`. 
The `@` indicates that `@title` is an argument that will be passed into the component, 
which makes it easier to reuse the same component in other parts of the app we are building.

Also, note that ember strictly follows `title case naming` style in templates, so defining 
`<PeopleList>` as `<peoplelist>` or `<peopleList>` won't work.

We've also renamed scientist to the more-generic person, decreasing the coupling of our component 
to where it's used.
```

**Comment by sourabhmandal** [comment]
Hi @ijlee2  I agree with your suggested changes and have made changes accordingly.

Additionally, I have added following lines to make it clear that ember uses only title cased components to save the hassle of figuring it out themselves, as the docs doesnt specify to use camel case only nor it shows any error on running. It simply doesnt work.

```
<!-- currently -->
Note that we've changed the title from a hard-coded string ("List of Scientists") to `{{@title}}`. 
The `@` indicates that `@title` is an argument that will be passed into the component, 
which makes it easier to reuse the same component in other parts of the app we are building.

We've also renamed scientist to the more-generic person, decreasing the coupling of our 
component to where it's used.

<!-- Changed -->
Note that we've changed the title from a hard-coded string ("List of Scientists") to `{{@title}}`. 
The `@` indicates that `@title` is an argument that will be passed into the component, 
which makes it easier to reuse the same component in other parts of the app we are building.

Also, note that ember strictly follows `title case naming` style in templates, so defining 
`<PeopleList>` as `<peoplelist>` or `<peopleList>` won't work.

We've also renamed scientist to the more-generic person, decreasing the coupling of our component 
to where it's used.
```

File: guides/release/getting-started/quick-start.md [hunk 1]
```diff
@@ -222,6 +222,8 @@ other parts of the app we are building.
 We've also renamed `scientist` to the more-generic `person`,
 decreasing the coupling of our component to where it's used.
 
+Also, note that ember strictly follows `title case naming` style in templates, so defining `<PeopleList>` as `<peoplelist>` or `<peopleList>` won't work.
```
Comment (ijlee2) [inline guides/release/getting-started/quick-start.md:225]: In the section [Introducing Components](https://guides.emberjs.com/release/components/introducing-components/), I found a "Zoey says" that advises readers to check casing. I liked that it drew a reader's attention, but found its message to be a bit unclear:

> introducing_components

Can we change your line to the following:

```suggestion
Our component is called `<PeopleList>`, based on its name on the file system. Please check that the letters P and L are capitalized.

<div class="cta">
  <div class="cta-note">
    <div class="cta-note-body">
      <div class="cta-note-heading">Zoey says...</div>
      <div class="cta-note-message">
        A component's name is derived from its file name.
        We capitalize the first letter and every letter after <code>-</code>, then remove the hyphens.
        This is known as pascal case.
      </div>
    </div>
    <img src="/images/mascots/zoey.png" role="presentation" alt="">
  </div>
</div>
```

I think these lines can address both your concern as well as mine.

If so, afterwards, will you update the "Zoey says" in the Introducing Component section to use the `<div>` code above?

File: guides/release/getting-started/quick-start.md [hunk 2]
```diff
@@ -244,7 +246,10 @@ Let's replace all our old code with our new componentized version:
     <li>{{scientist}}</li>
   {{/each}}
 </ul>
-<PeopleList @title="List of Scientists" @people={{@model}} />
+<PeopleList 
+  @title="List of Scientists" 
+  @people={{@model}} 
+  />
```
Comment (ijlee2) [inline guides/release/getting-started/quick-start.md:252]:
```suggestion
/>
```

File: guides/release/getting-started/quick-start.md (hunk 1 above)
Comment (sourabhmandal) [reply guides/release/getting-started/quick-start.md:225]: Your sentence is well worded and it does clarifies my concerns. I would be happy to help updating zoey says. Thankyou for the suggesting.

Comment (sourabhmandal) [reply guides/release/getting-started/quick-start.md]: Your sentence is well worded and it does clarifies my concerns. I would be happy to help updating zoey says. Thankyou for the suggestion

**Comment by ijlee2** [comment]
Ah, I saw #1520 afterwards. Look forward to it!

**Pull request merged**